lib/bloc/transactions/value_transfer/vtt_create/vtt_create_bloc.dart

```diff
@@ -307,7 +342,7 @@ class VTTCreateBloc extends Bloc<VTTCreateEvent, VTTCreateState> {
 
   /// add a [ValueTransferOutput] to the [VTTransaction].
   void _addValueTransferOutputEvent(
-      AddValueTransferOutputEvent event, Emitter<VTTCreateState> emit) {
+      AddValueTransferOutputEvent event, Emitter<VTTCreateState> emit) async {
```

Comment:
Does this function need to be async?
```suggestion
      AddValueTransferOutputEvent event, Emitter<VTTCreateState> emit) {
```